BLOG.md

```diff
@@ -0,0 +1,996 @@
+                custom_participant_id=user_email,
+            )
+            live_request = LiveVideoRequest.objects.create(
+                dyte_meeting_id=dyte_meeting["id"],
+                user_name=user_name,
+                user_email=user_email,
+                status=LiveVideoRequest.PENDING,
+                user_dyte_participant_id=participant["id"],
+                product=data["product"],
+            )
+            live_request_serializer = LiveVideoRequestSerializer(live_request)
+            return Response(
+                live_request_serializer.data, status=status.HTTP_201_CREATED
+            )
+
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+```
+This method creates a new instance of `LiveVideoRequest`. It also uses the `DyteAPIClient.create_meeting` and `DyteAPIClient.add_participant` to create a new Dyte meeting and add our user as a participant. You will shortly use this API to request a video session from our frontend application.
+
+#### 2. List all the pending requests
+
+Add the `list` method to `LiveVideoRequestViewSet`.
+```python
+    def list(self, request):
+        live_requests = LiveVideoRequest.objects.filter(status=LiveVideoRequest.PENDING)
+        serializer = LiveVideoRequestSerializer(live_requests, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+```
+We will use this API in support user's frontend to display a list of all the pending requests.
+
+#### 3. Fulfill the live shopping request
+
+Let's add a custom `action` to assign a support user to the live shopping request.
+
+```python
+    @action(methods=["post"], detail=True)
+    def start(self, request, pk=None):
+        live_request = get_object_or_404(LiveVideoRequest, pk=pk)
+        if live_request.support_user_dyte_participant_id:
+            token = DyteAPIClient.refresh_participant_token(
+                live_request.dyte_meeting_id,
+                live_request.support_user_dyte_participant_id,
+            )
+            return Response({"dyte_auth_token": token["token"]})
+        participant = DyteAPIClient.add_participant(
+            live_request.dyte_meeting_id,
+            "Customer Support",
+            "video_shoping",
+            "customer_support",
+        )
+        live_request.support_user_dyte_participant_id = participant["id"]
+        live_request.status = LiveVideoRequest.ACTIVE
+        live_request.save()
+        return Response(
+            {"dyte_auth_token": participant["token"]}, status=status.HTTP_201_CREATED
+        )
+```
+We are using the `DyteAPIClient.add_participant` to add a `Customer Support` participant to our Dyte meeting and returns `dyte_auth_token` for this participant.
+
+#### 4. Getting dyte participant token for user
+
+Now, let's add another custom action, `user_token`, to retrieve `dyte_auth_token` for the customer. We will use this API to instantiate DyteClient for user.
```

Comment:
did you plan to write instantiate here? or initiate?

Reply:
I meant to write instantiate. But initiate works too.